apache/knox
# KNOX-2852 - Bumped decode-uri-component in angular projects and schematics in webshell-ui

## What changes were proposed in this pull request?

The `@angular-eslint/schematics` dependency was upgraded from `1.2.0` to `15.1.0` in Webshell-UI.
The `decode-uri-component` in all angular projects was upgraded from `0.2.0` to `0.2.2`.

## How was this patch tested?

I have done the following manual tests on the UI:

Homepage:
* Tried the links
* Mouse on service name -> description shows up
* When UI services are clicked they navigate to UI URL
* When API services are clicked description shows up
* Dropdown menus open and close
* New topologies appear under the topologies menu

Token-generation:
* set server-managed to true and false
* set impersonation to true and false
* set lifespan.input.enabled to true and false
* tried to generate token over the max lifespan limit -> generate anyway/adjust lifetime
* generated tokens until reached token limit
* authenticated with a generated token

Token-management:
* generated normal token
* generated impersonated token
* disabled impersonation
* set server-managed to false
* disabled and revoked the created tokens

Admin-ui:
* Created, deleted, modified service definitions
* Created, deleted, modified provider configuration
* Created, deleted, modified descriptor
* Copied, deleted, modified topology

WebShell-UI:
* Configured webshell and link appeared on homepage
* Signed in to knox with the user tom (who is a user on my local computer) and was able to connect to my computer
